tests/test_cesm_fv.py

```diff
@@ -29,7 +30,7 @@ def retrieve_test_file():
         with open(p, "wb") as f:
             f.write(r.content)
 
-    return p
+    return str(p)
```

Comment:
Converting to str shouldn't be necessary. Did you run into an error?

Reply:
Oh right the path manipulation stuff. I updated some of the other similar ones to accept `Path`. Perhaps you could do that for this one too...

Reply:
For the quick fix, undo this but apply `str()` where you are passing `test_file_path` in `test_open_mfdataset`.

Reply:
I did, because the reader has a function called _ensure_mfdataset_filenames() that checks if the file is a string and, if not, attempts to directly sort it. I did not change that (although I'm not sure we really need that, since xarray's default error handling is quite clear if the format is not right IMHO). 
There are two alternative solutions:
- If you also think that the check is not really needed, I can remove it, and converting into a string wouldn't be necessary.
- If you'd rather keep the check as a safeguard, I could add the possibility of checking for a PosixPath. The only reason I did not go that way is that it implies importing another module, which seemed like unnecessary overhead.

Reply:
> Oh right the path manipulation stuff. I updated some of the other similar ones to accept `Path`. Perhaps you could do that for this one too...

Sorry, my computer didn't update that you already mentioned this. I'll check your fixes tomorrow and update this that way.

Reply:
I think this is now ok for now, we can always add `Path` support later.